CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 ## Unreleased
 
+Note: This release broadly applies some style guidelines across the repository. As a result,
+some classes that were visible might be package/private. Other non-final classes may now
+be final. See
+[#2182](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2182)
+and
+[#2210](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2210)
+and
+[#2212](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2212)
+and
+[#2213](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2212)
+for examples and details. These changes are not expected to be breaking, so please open
+an issue if this causes problems.
+
+### Baggage processor
+
+- Add baggage before it can be exported
+  ([#2152](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2152))
+- Add declarative config for baggage
+  ([#2031](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2031))
+
+### Disk buffering
+
+- Catching IllegalStateException in case of failed deserialization
+  ([#2157](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2157))
+- Apply final to public API classes where possible - disk-buffering
+  ([#2216](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2216))
+- Handle empty attribute values in disk buffering
+  ([#2268](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2268))
+
+### Inferred spans
+
+- Add dynamically changing the inferred span interval
+  ([#2153](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2153))
+
+### JMX scraper
+
+- Implement stable service.instance.id
+  ([#2270](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2270))
+
+### Kafka exporter
+
+- Add Kafka connectivity error handling
+  ([#2202](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2202))
+
+### OpAMP client
+
+- Move important user-facing classes out of 'internal' package.
```

Comment:
```suggestion
- Move important user-facing classes out of 'internal' package
```

Reply:
We had a conflicting standard here internally...so this is a hard one for me to break. I'll try! 😁